javascript/javascript1/week3/HomeWork/smartPhoneUsageApp.js

```diff
@@ -0,0 +1,92 @@
+console.log("\n*********** Smart Phone Activity Log ****************");
+const activities = [];
+const todayDate = getTodayDate();
+
+addActivity(todayDate, "LinkedIn", 50);
+addActivity(todayDate, "Google,", 5);
+addActivity(todayDate, "facebook", 25);
+addActivity(todayDate, "instagram", 36);
+addActivity(todayDate, "whatsapp", 1);
+showActivities();
+showStatus();
+mostTimeSpentOn();
+
+function getTodayDate() {
+  const currentDate = new Date();
+  const date = [
+    currentDate.getDate(),
+    currentDate.getMonth(),
+    currentDate.getFullYear(),
+  ];
+  const [day, month, year] = date;
+  const dateObj = new Date(`${year}-${month}-${day}`);
+  const options = { year: "numeric", month: "short", day: "numeric" };
+  return dateObj.toLocaleDateString("en-US", options).replace(",", "");
+}
+
+function addActivity(date, activity, duration) {
+  activities.push({
+    date: date,
+    activity: activity,
+    duration: duration,
+  });
```

Comment:
You can write this shorter when the keys are the same as the properties:
```suggestion
  activities.push({ date, activity, duration });
```